packages/default/scss/menu/_layout.scss

```diff
@@ -1,10 +1,17 @@
+$menu-separator-spacer: 2px !default;
+$image-spacing: 8px !default;
+
 @include exports("menu/layout") {
 
     // Base
     .k-menu {
         display: flex;
         flex-direction: row;
         align-items: stretch;
+    }
+
+    .k-menu,
+    .k-menu-scroll-wrapper {
```

Comment:
Styles placed inside this selector are used also for sub-items. When a menu is scrollable, its submenu container is not a child of k-menu. The animation container with sub-items is a sibling of the .k-menu.k-widget wrapper, placed inside the k-menu-scroll-wrapper. To prevent redundant rules, we can use ".k-menu-scroll-wrapper > .k-menu-group" selector instead of ".k-menu-scroll-wrapper" only. But in this way, the selector will use more class names and it will be necessary to use a stronger selector for overwriting these rules.

Reply:
I have added a new commit, with another option, which uses menu-group selector approach instead. https://github.com/telerik/kendo-themes/pull/127/commits/7100c2c2f3adc84f8e1c924bba43700c4c0e0a46